shell.py

```diff
@@ -168,6 +183,8 @@ def shell():
                 fullread()
             elif TEST_SCRIPT_1.startswith(command_param):
                 print(full_write_and_read_compare())
+            elif TEST_SCRIPT_1.startswith("2_"):
```

Comment:
이거는 
```suggestion
            elif TEST_SCRIPT_1.startswith(command_param):
```
이렇게 가야할거같아요.